README.md

```diff
@@ -1 +1,134 @@
-# enumerables
\ No newline at end of file
+# Enumerable Methods
+
+> The main goal of this project is to learn how to use Ruby to create Enumerable methods that are identical to the pre-defined methods.
+The following were the methods created
+
+- #my_each
+- #my_each_with_index
+- #my_select
+- #my_all?
+- #my_any?
+- #my_none?
+- #my_count
+- #my_map
+- #my_inject
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+No prerequisites
+
+To get a local copy up and running:
+
+1) Clone the repo or download the Zip folder
+2) Install ruby and run "ruby bubble_sort.rb" in your terminal
```

Comment:
Are you sure this instruction is for this project?

Reply:
>  #my_each when a block is given when self is a hash behaves like #each calls the given block once for each element in self
Not sure about this, it seems like I get exactly same output compared to original each :)
![image](https://user-images.githubusercontent.com/38778743/111875223-fac23d00-8998-11eb-8bcb-b2946e10790a.png)